x-pack/packages/kbn-entities-schema/package.json

```diff
@@ -2,5 +2,15 @@
   "name": "@kbn/entities-schema",
   "private": true,
   "version": "1.0.0",
-  "license": "Elastic License 2.0"
-}
\ No newline at end of file
+  "license": "Elastic License 2.0",
+  "scripts": {
+    "openapi:generate": "node scripts/generate_oas",
+    "openapi:serve": "node scripts/serve_oas_ui"
+  },
+  "dependencies": {
```

Comment:
I think we should put this in devDependencies as we don't need them at runtime

Reply:
looks like CI doesn't like that idea https://buildkite.com/elastic/kibana-pull-request/builds/227358#01914bf7-9065-4bb0-8ed8-19e823b63fed

Reply:
Fixed by lifting dependencies to root package.json